sophia-vote-contract.md

```diff
@@ -82,10 +88,10 @@ Since in Sophia ML we don not have `.count` or `.length` to get the length of a
 
 The `size` function is defined to accept a list of `'a` which is the convention for a generic type, and a return type `int` . In the function’s body we are calling the `size'` function while passing the list and an initial value for the counter.
 ``````
-private function size'(l : list('a), x : int) : int =
-      switch(l)
-         [] => x
-         _ :: l' => size'(l', x + 1)
+function size'(l : list('a), x : int) : int =
```

Comment:
With compiler v4.2.0 this can be written more nicely as:
```
function 
  size' : (list('a), int) => int
  size'([], x)     = x
  size'(_ :: l, x) = size'(l, x + 1)
```
But as @thepiwo mention it is a standard library function.

Reply:
Changes made. Can you resolve the conversation?

Reply:
still seems to be there in the explaination

Reply:
Yes. I want the whole contract reviewed before I change the content of the tutorial itself.

Reply:
I don't think there is any need to document `size` any more